.github/workflows/deploy-full-stack.yml

```
@@ -0,0 +1,406 @@
+        run: |
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            echo "backend-changed=${{ github.event.inputs.deploy_backend }}" >> $GITHUB_OUTPUT
+            echo "frontend-changed=${{ github.event.inputs.deploy_frontend }}" >> $GITHUB_OUTPUT
+          else
+            # Check for changes in backend files
+            if git diff --name-only HEAD~1 HEAD | grep -E "(apps/express-api|infrastructure)" > /dev/null; then
+              echo "backend-changed=true" >> $GITHUB_OUTPUT
+            else
+              echo "backend-changed=false" >> $GITHUB_OUTPUT
+            fi
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)